test/database.ts

```diff
@@ -295,43 +327,30 @@ describe('Database', () => {
       assert(database.formattedName_, formattedName);
     });
 
-    it('should create a SessionPool object', () => {
-      assert(database.pool_ instanceof FakeSessionPool);
-      assert.strictEqual(database.pool_.calledWith_[0], database);
-      assert.strictEqual(database.pool_.calledWith_[1], POOL_OPTIONS);
-    });
```

Comment:
Why do we need to delete this test case, database class still has "pool_" object. Can we modify this test to use sessionfactory to instantiate pool object

Reply:
Now, the creation of session pool object is happening in the SessionFactory class, hence I have added a test in the tests for Session Factory and we need to delete this from here